src/main/java/org/cubeville/cvgames/models/QueueSign.java

```diff
@@ -45,7 +48,17 @@ public void displayStatus(ArenaStatus status) {
 	}
 
 	public void displayFill() {
-		this.sign.setLine(1,"§l" + arena.getQueue().size() + "/" + arena.getQueue().getMaxPlayers());
+		if (arena.getStatus() != ArenaStatus.IN_USE || arena.getQueue().getGame() == null) {
```

Comment:
Is there a case where the arena is in use and the game is null? If there isn't, then I don't think we need the check to see if the game is null or not.